src/client/actions.ts

```diff
@@ -0,0 +1,133 @@
+/**
```

Comment:
This gives a bit of an overview of what actions are and how they work.

Kinda important that this part makes sense, so please nit-pick!